target/linux/econet/base-files/sbin/en75_chboot

```diff
@@ -99,32 +101,32 @@ switch() {
 }
 
 main() {
-    machine=$(sed -n -e 's/^machine\s\+:\s\+//p' < /proc/cpuinfo)
-    if [ "$machine" = "TP-Link Archer VR1200v (v2)" ]; then
+    machine=$(board_name)
+    if [ "$machine" = "tplink,archer-vr1200v-v2" ]; then
```

Comment:
Hi! It would be better if you also replace this if-elif-else sequence with a switch/case statement, like how other scripts (e.g. [this one](https://github.com/openwrt/openwrt/blob/main/target/linux/ath79/generic/base-files/etc/board.d/02_network) ) handles board-level differences.